providers/azure/aci.go

```diff
@@ -713,3 +719,37 @@ func aciContainerStateToContainerState(cs aci.ContainerState) v1.ContainerState
 		},
 	}
 }
+
+// Filters service account secret volumes for Windows.
+func filterServiceAccountSecretVolume(osType string, containerGroup *aci.ContainerGroup) {
+	if strings.EqualFold(osType, "Windows") {
+		serviceAccountSecretVolumeName := ""
```

Comment:
should it be an array?

Reply:
Will there be multiple service accounts?